Bottom: d6c7e53dd3bcd641b725ee689dcd8fe486e32861
Top:    1d5006aadc74ad1ef44c5f5a451e8a0c1957dafb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 19:43:00 +0000

Refresh of add-flag-parameters-to-resetfo

---

diff --git a/src/find.c b/src/find.c
index 32fe7fe..ffbde1f 100644
--- a/src/find.c
+++ b/src/find.c
@@ -284,7 +284,7 @@ typedef struct
  * some local identifiers
  */
 static Coord Bloat = 0;
-/*static */int TheFlag = FOUNDFLAG;
+static int TheFlag = FOUNDFLAG;
 static int OldFlag = FOUNDFLAG;
 static void *thing_ptr1, *thing_ptr2, *thing_ptr3;
 static int thing_type;
